packages/toast/src/use-toast.tsx

```diff
@@ -202,15 +202,20 @@ export function createStandaloneToast({
  */
 export function useToast(options?: UseToastOptions) {
   const { theme, setColorMode, toggleColorMode, colorMode } = useChakra()
-  return React.useMemo(() => {
-    return createStandaloneToast({
-      theme,
-      colorMode,
-      setColorMode,
-      toggleColorMode,
-      defaultOptions: options,
-    })
-  }, [theme, setColorMode, toggleColorMode, colorMode, options])
+
+  const toastOptions = React.useRef(options)
+
+  return React.useMemo(
+    () =>
+      createStandaloneToast({
+        theme,
+        colorMode,
+        setColorMode,
+        toggleColorMode,
+        defaultOptions: toastOptions,
+      }),
+    [theme, setColorMode, toggleColorMode, colorMode, toastOptions],
```

Comment:
```suggestion
    [theme, setColorMode, toggleColorMode, colorMode],
```

Reply:
I pulled this branch down and built it again with `useLatestRef` hook. Typescript yelled at me if I omitted this from the dependency array.